src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx

```diff
@@ -63,12 +63,28 @@ class CreateKnownSchemeProposal extends React.Component<IProps, IState> {
     const actions = props.genericSchemeInfo.actions();
     this.state = {
       actions,
-      currentAction:  actions[0],
+      currentAction: actions[0],
       tags: new Array<string>(),
     };
   }
+  
+  private async getBountyEth(values: IFormValues): Promise<string> {
+    const currentAction = this.state.currentAction;
+    let ethToSend = 0;
+
+    for (const field of currentAction.getFields()) {
+      if (field.name === "_depositAmount" || field.name === "_amount" && this.props.genericSchemeInfo.specs.name === "Standard Bounties") {
+        ethToSend += parseInt(field.callValue(values[field.name]))/1000000000;
```

Comment:
Basically, all monetary values in the app are `BN` instances in the smallest unit of account of the token. So, in the case of ETH, the values hould be converted to WEI as soon as possible, and this is the right place.

Edson, please do not implement your own logic to handle this, but use the `utils.toWEI` helper function, which will give you exactly what you need and we want here :-)  (and it will also avoid the integer errors that you undoubtedly ran into with this code)

Reply:
In 8fa6cde value is already Wei. However, BN.js is used for addition